### ISSUE TITLE
Development Enviroment: How does the search work and where to add Algolia API Keys?

### ISSUE THREAD

**Issue by donenoyes** [sendgrid/docs#6513]
## Expected content

Find a config file to add Algolia API Keys and index

## Actual content

No way to add Algolia API Keys as far as I know. 

## Additional Context

My understanding is that we need Algolia API keys and index and other keys to add in config file to make algolia work.

But I am not able to find such file. Could you please help with the issue?

When using the docs locally after npm install and develop and trying to search, it works for existing content.

But when trying to add new content, the search does not include it.
I also noticed, this project does not use algolia gatsby plugin but has "search by algolia" under search bar.

TLDR: How the search is working and how can I make sure my new content is indexed by the search in this gatsby project?

**Comment by Whatthefoxsays** [comment]
Hi @donenoyes, 

I'm not sure what you're trying to accomplish exactly, but this is a documentation repository, not an Algolia repository. We use Algolia for our search, but we don't control any of the functionality from this repo. You can find more information on their products [here](https://www.algolia.com/).
